channel.go

```diff
@@ -43,6 +43,9 @@ var (
 
 const (
 	ephemeralHostPort = "0.0.0.0:0"
+
+	// DefaultTraceSampleRate is the default sampling rate for traces.
+	DefaultTraceSampleRate = 1.0
```

Comment:
Yes, it's also 1 in Python, but 0.01 in Node.  There are two sides of the coin:
1. for any service with reasonable traffic, the 100% sampling can easily overwhelm the collectors, 
2. for low-traffic services the `0.01` setting might produce no traces. 

Of those two alternatives it's better to be on the safe side. Most people will go to prod without changing the default.